### PR TITLE
Update image-streams-rhel7.json

### PR DESCRIPTION
**Pull request by andrewklau**
Fix the notices for latest images in rhel image streams

### CODE REVIEW

**Comment by bparees** [comment]
needs hack/update-generated-bindata.sh run

**Comment by bparees** [comment]
@andrewklau if you don't have a dev environment in place to generating the bindata file, let me know and i can re-do this PR w/ the necesary update.

also feel free to squash your other PR https://github.com/openshift/origin/pull/11800 into this one, it'll make it faster to get merged (and you'll only have to generate a single bindata and don't have to worry about conflicting w/ yourself)

File: examples/image-streams/image-streams-rhel7.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
             "name": "latest",
             "annotations": {
               "openshift.io/display-name": "Node.js (Latest)",
-              "description": "Build and run Node.js applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/4/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Ruby available on OpenShift, including major versions updates.",
+              "description": "Build and run Node.js applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/4/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major versions updates.",
```
Comment (bparees) [inline examples/image-streams/image-streams-rhel7.json]: @luciddreamz :(

**Comment by andrewklau** [comment]
@bparees I'll give it a shot soon otherwise I'll merge them and ping you, thanks!

**Comment by andrewklau** [comment]
@bparees done

**Comment by bparees** [comment]
lgtm, thanks @andrewklau!
[merge]

**Comment by bparees** [comment]
flake https://github.com/openshift/origin/issues/11662
[test]

**Comment by bparees** [comment]
[test]

**Pull request merged**